### PR TITLE
Focus Avo functions 101 docs on the destination interface

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

File: pages/implementation/avo-functions-overview.mdx [hunk 1]
```diff
@@ -179,27 +184,22 @@ object properties: exact structure of the allowed keys and types of values in th
...
-The last step is actually sending the provided data to all the configured tracking destinations. There are different classes of events that can be tracked: identify, log event, update user properties, unidentify, log revenue, log page view, but everything is handled inside the Avo file and delivered in a correct form to your destinations. If you are using a Custom Destination the data will be provided to the callback object you supplied during initialization.
+Avo makes sure that the event names and provided properties are correct according to the tracking plan with compile time and runtime validations, development invocations are reported to Avo servers so you can check the invocation status on the Avo website.
+The last step is actually sending the provided data to all the configured tracking destinations. There are different classes of events that can be tracked: identify, log event, update user properties, unidentify, log revenue, log page view,
```
Comment (logason) [inline pages/implementation/avo-functions-overview.mdx:188]:
```suggestion
The last step sends the provided data to all the configured tracking destinations. There are different classes of <Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> that can be tracked: identify, log event, update user properties, unidentify, log revenue, log page view, 
```

File: pages/implementation/avo-functions-overview.mdx [hunk 2]
```diff
@@ -179,27 +184,22 @@ object properties: exact structure of the allowed keys and types of values in th
...
+Avo makes sure that the event names and provided properties are correct according to the tracking plan with compile time and runtime validations, development invocations are reported to Avo servers so you can check the invocation status on the Avo website.
+The last step is actually sending the provided data to all the configured tracking destinations. There are different classes of events that can be tracked: identify, log event, update user properties, unidentify, log revenue, log page view, 
+but everything is handled inside the Avo file and delivered in a correct form to your destinations. The data will be provided to the Destination Interface methods you've defined during initialization.
 Tracking code would look similar to this:
```
Comment (logason) [inline pages/implementation/avo-functions-overview.mdx:190]:
```suggestion
The Avo generated tracking code will look similar to this:
```

File: pages/implementation/devs-101.mdx [hunk 3]
```diff
@@ -27,18 +27,18 @@ analytics implementation you have to revisit to fix again and again and again 
...
 
-The Avo Functions have a <Link passHref href="/implementation/destinations"><a>Destination Interface</a></Link> 
-that you have full control over how you implement. You provide a simple Destination Interface 
-object when initializing Avo Functions SDK, so that any time an Avo Function calls an analytics 
-logging method (e.g. logEvent, setUserProperties, etc), you can govern exactly what analytics 
-SDK method should be called.
+When designing the tracking plan you configure the list of analytics destinations for each source. In the generated code each destination is 
+represented as its own Destination Interface in the initAvo method of your Avo Functions. This is how Avo Functions routes data, 
+locally in your client (without the data ever going through Avo's servers), to the correct destination based on the source/destination connections 
+you've defined in the Avo dashboard.
```
Comment (logason) [inline pages/implementation/devs-101.mdx:37]: Some weird newlines here, is this intentional, or some formatter messing around 👀  Probably doesn't have impact on the output right?

File: pages/implementation/avo-functions-overview.mdx [hunk 4]
```diff
@@ -7,11 +7,15 @@ import Link from '../../components/Link';
...
 
-The Avo file is the code representation of the tracking plan you define in Avo for a given source. You can think of a source as a platform that sends events. For example if you set up an iOS/Swift source the generated file would be a `.swift` file. Since the tracking plans are different for each customer the content of this file is also different. We generate the files and you can download them either with <Link passHref href="/implementation/cli"><a>the Avo CLI</a></Link> or in the Avo web interface in <Link passHref href="/workspace/implement"><a>the Implement tab</a></Link>.
+The Avo generated code is the representation of the tracking plan you define in Avo for a given source. You can think of a source as a platform that sends events. 
+For example if you set up an iOS/Swift source the generated code will be in a `.swift` file. 
+Since the tracking plans are different for each customer the content of this file is also different. 
+We generate the files and you can download them either with <Link passHref href="/implementation/cli"><a>the Avo CLI</a></Link> or in the Avo web interface in <Link passHref href="/workspace/implement"><a>the Implement tab</a></Link>.
```
Comment (logason) [inline pages/implementation/avo-functions-overview.mdx:13]: Any change here other than the line breaks? 👀  Don't we want to keep this in a single paragraph?

**Review by logason — APPROVED** [review]
Few minor suggestions, otherwise lgtm!

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…avo-functions-overview**](https://docs-fwkdh50j6.avo-dev.app/implementation/avo-functions-overview) | [**/implementation/devs-101**](https://docs-fwkdh50j6.avo-dev.app/implementation/devs-101) |
|:-:|:-:|
|<a href="https://docs-fwkdh50j6.avo-dev.app/implementation/avo-functions-overview">Screenshot of /implementation/avo-functions-overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/UecFajNqdiK4mPtIOoX4rzoZmzS3nSdAWO9XRb5RV3gV5_HBsK34KaYFhf-Do4zzTM0GIeu7QBIDHIz5Rg3Fuw.png">(view full size)</a> |<a href="https://docs-fwkdh50j6.avo-dev.app/implementation/devs-101">Screenshot of /implementation/devs-101</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/YeeEXJCWq2IdEgHDqgkbNUiPLHRHxWeQyxx_vNy9gfEc2Cke8L_fEz3esm2N5IfNgM4I5yt1law0FIQC9lIw8Q.png">(view full size)</a> |







Commit 1b5e1838293063ee5545e4591783a2b016ca4e19 (https://docs-fwkdh50j6.avo-dev.app).

File: pages/implementation/avo-functions-overview.mdx (hunk 4 above)
Comment (alex-tpom6oh) [reply pages/implementation/avo-functions-overview.mdx:13]: Yes, that's only for readability in the editor. Single line break does not change the output, to get a new paragraph we need a double line break

File: pages/implementation/devs-101.mdx (hunk 3 above)
Comment (alex-tpom6oh) [reply pages/implementation/devs-101.mdx:37]: Just formatting for the code editor, does not affect the output

File: pages/implementation/devs-101.mdx [hunk 5]
```diff
@@ -27,18 +27,18 @@ analytics implementation you have to revisit to fix again and again and again 
...
 
-In the <Link passHref href="/workspace"><a>Avo Tracking Plan</a></Link> you can configure which analytics <Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> should be triggered with an Avo Function (for example you might want to update user properties or log revenue along with logging an event), and the Avo Functions 
+In the <Link passHref href="/workspace"><a>Avo Tracking Plan</a></Link> you can configure which analytics 
+<Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> should be triggered with an Avo Function (for
```
Comment (alex-tpom6oh) [inline pages/implementation/devs-101.mdx:40]: The only case when a new line is actually applied is when a link is the first thing on the new line, like here 😬

**Pull request merged**